css/style.css

```diff
@@ -14,3 +14,33 @@ a:hover {
 h1 {
     font-size: 2em;
 }
+
+.tags {
+    list-style: none;
+    margin: 0;
+    overflow: hidden;
+    padding: 0;
+}
+
+.tags li {
+    float: left;
+}
+
+.tag {
+    background: #eee;
+    border-radius: 3px;
+    color: #777;
```

Comment:
Let's try to keep the colors consistent here. Maybe use #159957, the color used for the titles, or anything else that you think would fit well.